Mathlib/GroupTheory/OrderOfElement.lean

```diff
@@ -188,14 +188,17 @@ theorem pow_orderOf_eq_one (x : G) : x ^ orderOf x = 1 := by
 theorem orderOf_eq_zero (h : ¬IsOfFinOrder x) : orderOf x = 0 := by
   rwa [orderOf, minimalPeriod, dif_neg]
 
-@[to_additive]
+@[to_additive (attr := simp)]
```

Comment:
I am not so convinced about the `simp` attribute here: I can imagine that very often you have some formula for `orderOf` in context that you plan to use to `rw` the `orderOf` in the goal, but after you "`simp`-ed out" some trivialities.

Now, you also have to remove this lemma from the `simp` context.